### DISCUSSION TITLE
npx ava -r dotenv-flow/config not working

### DISCUSSION THREAD

**Discussion by sombriks** [avajs/ava#3256] [Q&A] [answered]
Hello! I want to load environment-specific variables with the help of a dotenv library.

My current workflow with other libraries is to use the '-r <module>' argument so node car preload the library for me so it populates my environment variables for me in a transparent way.

**Comment by novemberborn** [comment, marked as answer]
`-r` is a Node.js argument, not an AVA argument. You can load `dotenv-flow/config` in test files using the `require` config, e.g. `ava.config.mjs`:

```
export default {
  require: ['dotenv-flow/config']
}
```

**Reply by sombriks** [reply]
oh i must have missed the docs about this and just tried to mimic what mocha does.
thanks for the guidance!

**Comment by sombriks** [comment]
[this is what i've missed](https://github.com/avajs/ava/blob/main/docs/06-configuration.md).

and [this](https://github.com/sombriks/sample-testable-code/blob/main/04-dependency-injection-points/package.json#L30) is a nice sample with the package.json alternative config :-)

i feel that i can fully migrate from mocha to ava now, thank you very much!
